Llama.cpp Typescript: Cannot find name 'LoadModel'

There is an error when compiling Typescript code using Llama.cpp:

```
$ tsc -p .
node_modules/@llama-node/llama-cpp/index.d.ts:137:31 - error TS2304: Cannot find name 'LoadModel'.

137   static load(params: Partial<LoadModel>, enableLogger: boolean): Promise<LLama>
```

I found that changing in `llama-cpp/index.d.ts` line 137 `static load(params: Partial<LoadModel>` by `static load(params: Partial<ModelLoad>` works. It is probably a typo. 
I found another reference to `LoadModel` in `llama-cpp/src/lib.rs`, so unfortunately as I don't know Rust I can not suggest a valid fix in a PR.

--- a/packages/llama-cpp/index.d.ts
+++ b/packages/llama-cpp/index.d.ts
@@ -134,7 +134,7 @@ export interface LlamaLoraAdaptor {
   nThreads: number
 }
 export class LLama {
-  static load(params: Partial<LoadModel>, enableLogger: boolean): Promise<LLama>
+  static load(params: Partial<ModelLoad>, enableLogger: boolean): Promise<LLama>
   getWordEmbedding(params: Generate): Promise<Array<number>>
   tokenize(params: string): Promise<Array<number>>
   inference(params: Generate, callback: (result: InferenceResult) => void): () => void

--- a/packages/llama-cpp/src/lib.rs
+++ b/packages/llama-cpp/src/lib.rs
@@ -32,7 +32,7 @@ pub struct LLama {
 impl LLama {
     #[napi]
     pub async fn load(
-        #[napi(ts_arg_type = "Partial<LoadModel>")] params: serde_json::Value,
+        #[napi(ts_arg_type = "Partial<ModelLoad>")] params: serde_json::Value,
         enable_logger: bool,
     ) -> Result<LLama> {
         let params = serde_json::from_value::<ModelLoad>(params).unwrap();

